### PR TITLE
Add versioned graphql types via extension instead of direct YAML

### PR DESCRIPTION
**Pull request by unclecheese**
In adherence to new multi-schema work.

Resolves: https://github.com/silverstripe/silverstripe-graphql/issues/158

### CODE REVIEW

File: src/GraphQL/Extensions/ManagerExtension.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+use SilverStripe\Versioned\GraphQL\Types\VersionedStatus;
+
+class ManagerExtension extends Extension
+{
+    /**
+     * Adds the versioned types to all schemas
+     *
+     * @param $config
+     */
+    public function updateConfig(&$config)
```
Comment (maxime-rainville) [inline src/GraphQL/Extensions/ManagerExtension.php]: Any way we could had a bit of logic to test those extra types are added correctly?

Comment (maxime-rainville) [reply src/GraphQL/Extensions/ManagerExtension.php]: Good enough. I withdraw my pedantic objection.

**Comment by maxime-rainville** [comment]
We need silverstripe/silverstripe-graphql#173 and silverstripe/recipe-cms#18 to be merged first.

Then we can re-trigger the build.

**Comment by robbieaverill** [comment]
Dependencies merged and builds restarted

**Pull request merged**